src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(function(resolve, reject) {
+  resolve(logo.addEventListener('click', () => {
+    logo.insertAdjacentHTML('afterend', `
+      <div class="message">Promise was resolved!<div>
+    `);
+  }));
+});
+
+promise1
+  .then(
+    result => result
```

Comment:
```suggestion
    onSuccess
```

Reply:
create onSuccess handler with logic related to show success message. Make the same with second promise